.github/actions/passwords-tool/script.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
```

Comment:
Change the file name to `tests-stack.sh`

Reply:
Done https://github.com/wazuh/wazuh-packages/pull/1737/commits/054ac37040357937607329818f7f2548254c86cc